runtime/centrifuge/src/lib.rs

```diff
@@ -1666,7 +1739,24 @@ pub type UncheckedExtrinsic =
 /// Extrinsic type that has already been checked.
 pub type CheckedExtrinsic = generic::CheckedExtrinsic<AccountId, RuntimeCall, SignedExtra>;
 
-type UpgradeCentrifuge1019 = pallet_loans_ref::migrations::v1::Migration<Runtime>;
+parameter_types! {
+	// = 16.65 CFG per epoch (12h)
+	// https://gov.centrifuge.io/t/rfc-add-a-block-reward-and-improve-collators-cycle/4766
+	pub const CollatorRewards: Balance = 8_325 * MILLI_CFG;
+	// = 20,096 CFG per epoch (12h)
+	pub const TotalRewards: Balance = 10_048 * CFG;
```

Comment:
Why here are we not multiplying this by `MaxCandidates` as in Altair?

Reply:
In Altair, we transfer the rewards from the treasury and do not want to mint extra rewards into the treasury as the latter is already sufficiently funded. So the multiplication with `MaxCandidates` serves there as a max bound. The remainder of `TotalRewards - CollatorRewards` will be dropped in Altair.

However, in Centrifuge we mint rewards both for the collators as well as the treasury. These values have been pre-approved by https://gov.centrifuge.io/t/rfc-add-a-block-reward-and-improve-collators-cycle/4766